main.tf

```diff
@@ -496,3 +497,10 @@ resource "time_sleep" "wait_for_aws_s3_bucket_settings" {
   create_duration  = "30s"
   destroy_duration = "30s"
 }
+
+resource "time_sleep" "wait_for_aws_s3_ownership_change" {
```

Comment:
From my personal experience fixing this issue locally doesn't require the use of a `time_sleep` resource, instead, we can make it work by adding a `depends_on` like on #175, #176 and #177.

Reply:
#175 fixes with depends on

Reply:
That's fine with me if the depends_on works. I thought I had tried that locally to fix the issue but apparently not.